src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator/Calculator';
 
 function App() {
```

Comment:
- [ ] As stated in the instructions, your components should be class-based. Kindly change this functional **`App.js`** component to a class-based one.

Reply:
Done.